roles/openshift_health_checker/library/docker_container.py

```diff
@@ -0,0 +1,2036 @@
+#!/usr/bin/python
+# pylint: skip-file
+# flake8: noqa
```

Comment:
@juanvallejo FYI this line disable "[flake8](https://pypi.python.org/pypi/flake8)" (checks formatting and some common problems), while the other one disables "[pylint](https://pylint.readthedocs.io)". The two tools have some overlap, but also some unique features. We happen to use both for now.